hub/auto/unstructured/image_classification.py

```diff
@@ -146,10 +167,8 @@ def structure(  # type: ignore
                     ds[images_tensor_map[set_name]].append(reshaped_image)
 
                 except Exception as e:
-
-                    warnings.warn(
-                        f"[Skipping] Could not upload sample '{file_path}'. Reason: {e}"
-                    )
+                    skipped_files.append(file_path.name)
+                    ingestion_summary(str(self.source), skipped_files)
```

Comment:
can we modify the progress bar to have the len(skipped_files)?

Reply:
the pbar would look like this:


ingesting "my_dataset_folder" (0 files skipped) [|||||||||||||||||||||||||||xxxxxxxxx]

Reply:
https://stackoverflow.com/questions/37506645/can-i-add-message-to-the-tqdm-progressbar